docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,628 @@
+women aged 21 to 65; linear ramp up HPV vaccination to cover 90% of the women 
+aged 15 to 45 in 2030, then stay 90% in 2031-2040. Where:
+ - Women without HPV infection are provided with cytology every three years
+ - Women with HPV infection are provided cytology every year
+
+**Alternative scenario:** by 2040, project existing level of HPV vaccination for 
+women aged 15 to 45; linear ramp up cervical cancer screening to cover 70% of the 
+women aged 21 to 65 in 2030, then stay 70% in 2031-2040. Where:
+ - Women without HPV infection are provided with cytology every three years
+ - Women with HPV infection are provided cytology every year
+
+.. image:: cervical_cancer_scale_up.PNG
+
+.. list-table:: Intervention scale-up
+   :header-rows: 1
+
+   * - Scenario
+     - Intervention
+     - Year
+     - Coverage
+   * - Baseline
+     - Cervical cancer screening
+     - 2020-2040
+     - 25%
+   * - Baseline
+     - HPV vaccination
+     - 2020-2030
+     - Stay 10% in 2020-2021, then linearly ramp up from 10% to 90% in 2021-2030.
+   * - Baseline
+     - HPV vaccination
+     - 2031-2040
+     - 90%
+   * - Alternative
+     - HPV vaccination
+     - 2020-2040
+     - 10%
+   * - Alternative
+     - Cervical cancer screening
+     - 2020-2030
+     - Stay 25% in 2020-2021, then linearly ramp up from 25% to 70% in 2021-2030.
+   * - Alternative
+     - Cervical cancer screening
+     - 2031-2040
+     - 70%
+
+.. note::
+
+ - Wang et al. reported a current cervical cancer screening coverage of 20.7% 
+   with 95%CI 18.6-22.8 in China. We set it as 25% as we believe insured population has higher screening coverage than general population. 
+ - No data has identified for current HPV vaccination rates in China. Temporarily 
+   we will use 10%.
+ - The target screening and HPV vaccination coverage in 2030 are guided by WHO 
+   cervical cancer elimination strategy.
+
+
+.. _5.0:
+
+5.0 Vivarium modelling components
++++++++++++++++++++++++++++++++++
+
+.. _5.1:
+
+5.1 Vivarium concept model 
+--------------------------
+
+.. image:: cervical_cancer_concept_model_diagram.svg
+
+.. _5.2:
+
+5.2 Demographics
+----------------
+
+.. _5.2.1:
+
+5.2.1 Population description
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ - Cohort type: Closed cohort of 200,000 insured female (100%) simulants.
+ - Age and sex: Age 15 to 95+, 5 year-age bands, uniformly distributed age and 
+   sex structure.
+ - Time span: Jan 1, 2020 to Dec 31, 2040 with 36.5-day time-steps.
+ - Location: blended with province-specific weight in China.
+
+.. _5.2.2:
+
+5.2.2 Location description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provinces to model include Tianjin, Jiangsu, Guangdong, Henan, and Heilongjiang. 
+The uniform distribution of age and sex structure will be used among the different 
+provinces.
+
+.. list-table:: location weight table
+   :header-rows: 1
+
+   * - Province
+     - location id
+     - Weight
+     - Weighted ACMR (per 100,000 person-years)
```

Comment:
These tables are going off of the screen for me. I am not sure what the best alternative is, but just thought I would let you know!

Reply:
I noticed that issue, the simplest solution is to shrink the screen percent fit to your window.